angular-ngrx-scss/src/app/auth/auth.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component } from '@angular/core';
+import { FormControl, FormGroup } from '@angular/forms';
+import { AuthService } from './services/auth.service';
+
+@Component({
+  selector: 'app-auth',
+  templateUrl: './auth.component.html',
+  styleUrls: ['./auth.component.scss'],
+})
+export class AuthComponent {
+  signInGroup = new FormGroup({
+    csrfToken: new FormControl(''),
+    callbackUrl: new FormControl(''),
+  });
+
+  constructor(private authService: AuthService) {}
+
+  onSubmit() {
+    this.authService.signIn().subscribe(({ redirectUrl }) => {
+      window.location.href = redirectUrl;
+    });
```

Comment:
If we want this app to showcase NgRx I think we should move the auth flow to the NgRx effects. But we can do it while redoing this solution after BE is modified

Reply:
Fair point - adding it to my list of things to update when we do the OAuth fix.